custom_components/localtuya/climate.py

```diff
@@ -340,22 +432,14 @@ def min_temp(self):
         """Return the minimum temperature."""
         if self.has_config(CONF_MIN_TEMP_DP):
             return self.dps_conf(CONF_MIN_TEMP_DP)
-        # DEFAULT_MIN_TEMP is in C
-        if self.temperature_unit == UnitOfTemperature.FAHRENHEIT:
-            return DEFAULT_MIN_TEMP * 1.8 + 32
-        else:
-            return DEFAULT_MIN_TEMP
+        return self._config[CONF_TEMP_MIN]
```

Comment:
Not 100% sure about removing this conversion. I'm thinking that the CONF_TEMP_MIN is set in Fahrenheit during configuration if needed and conversion here is not needed. The same goes for CONF_TEMP_MAX of course.